components/BlogFeed.js

```diff
@@ -35,7 +35,17 @@ export default function BlogFeed({ posts }) {
       <div className="blog blog__container">
         <h1 className="blog__heading">TJI Blog</h1>
         <div className="blog__tagline">
-          <p>Where the TJI team posts blogs about our data, publications, analyses, and code.</p>
+          <p>
```

Comment:
@judan I just wanted to confirm that we want this updated blog text as part of this PR.

Reply:
I don't think we do want it based on what Eva said in our last meeting. I'll revert this for now. Thanks!